security/backends/elasticsearch/logstash/handler_tcp.py

```diff
@@ -12,7 +20,19 @@ class TCPLogstashHandler(SocketHandler):
 
     def __init__(self, host, port=5959):
         super().__init__(host, port)
-        self.formatter = LogstashFormatter()
```

Comment:
the formatter is not necessary and now we can use base formatter to change message (for example with prefix)